### PR TITLE
125 176 saving private calendar

### PR DESCRIPTION
**Pull request by bochka123**
image
image

### CODE REVIEW

**Comment by Anatolii-Ivanov** [comment]
Can you align text as in email field?

File: frontend/src/app/shared/components/calendar/calendar.component.sass [hunk 1]
```diff
@@ -20,5 +22,35 @@
...
+
+::ng-deep.mat-mdc-form-field-input-control
+    margin-top: 20px
+
+::ng-deep.mat-mdc-form-field-infix
+    min-height: 1px !important
+    height: 36px !important
+
+::ng-deep.mat-mdc-floating-label
+    top: 20px !important
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/calendar/calendar.component.sass:55]: Have you done it according to design or just make it work?

Comment (bochka123) [reply frontend/src/app/shared/components/calendar/calendar.component.sass:55]: image
image
Without it label in placeholder will be down

**Comment by bochka123** [comment]
image

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by KaterynaHerasymiva — APPROVED** [review]

**Pull request merged**